### DISCUSSION TITLE
Mouse DoubleClick

### DISCUSSION THREAD

**Discussion by Warsinger** [hajimehoshi/ebiten#3063] [General]
Is there a way to detect a mouse button double clicked? I'm using `inpututil.IsMouseButtonJustPressed` but I can't find anything about a double press. I have an idea of how to implement it manually but if someone's already done it I'd rather use that.
Thanks

**Comment by tinne26** [comment]
No, there's no built-in method, and the most robust input handling library for Ebitengine, https://github.com/quasilyte/ebitengine-input, doesn't include this on its methods either. It's quite uncommon to need double click in games, though.

If you are working on something small, just do it yourself, the implementation is trivial. If you are doing something more complex, you would need an input library abstraction anyway, so you could either request the feature on your existing library of choice, or implement it more robustly yourself.
